src/AzureFunctionPackage/Functions/Common/config.xml

```diff
@@ -62,7 +63,8 @@
     <merge from="release/2.1.5xx" to="release/2.2.1xx" />
     <merge from="release/2.2.1xx" to="release/2.2.2xx" />
     <merge from="release/2.1.6xx" to="release/2.2.2xx" />
-    <merge from="release/2.2.2xx" to="master" />
+    <merge from="release/2.2.2xx" to="release/3.0.1xx" />
+    <merge from="release/3.0.1xx" to="master" />
```

Comment:
Can you add toolset and core-sdk repos with just 3.0 -> master as well?

Reply:
Pushed.